src/openrct2/actions/TrackRemoveAction.hpp

```diff
@@ -406,7 +406,7 @@ DEFINE_GAME_ACTION(TrackRemoveAction, GAME_COMMAND_REMOVE_TRACK, GameActionResul
 
             if (entranceDirections & TRACK_SEQUENCE_FLAG_ORIGIN && (tileElement->AsTrack()->GetSequenceIndex() == 0))
             {
-                if (!track_remove_station_element(mapLoc.x, mapLoc.y, mapLoc.z / 8, _origin.direction, rideIndex, 0))
+                if (!track_remove_station_element({ mapLoc.x, mapLoc.y, mapLoc.z, _origin.direction }, rideIndex, 0))
```

Comment:
One last issue you can still use the other constructors to reduce having to get the components individually. Same applies to the other callers